docs/requirements.txt

```
@@ -0,0 +1,3 @@
+mkdocs==1.*
+pymdown-extensions==10.*
+mkdocs-material==9.*
```

I've had less issues with

```suggestion
mkdocs>=1,<2
pymdown-extensions>=10,<11
mkdocs-material>=9,<10
```

I'd prefer even more not adding upper bounds at all:

```suggestion
mkdocs>=1.5
pymdown-extensions>=10
mkdocs-material>=9
```

---

Why, though? I understand your first suggestion to be the same as mine, and the second suggestion to be quite dangerous, given how semantic versioning (which most packages adhere to nowadays) works.

---

The first suggestion is made for the reason mentioned in this previous PR: https://github.com/mkdocstrings/vba/pull/14.

The second suggestion is made for the reasons mentioned in this post: https://iscinumpy.dev/post/bound-version-constraints/. Basically, semver does not protect you, and upper bounds easily cause conflicts in the Python ecosystem, where packages can't be installed in multiple different versions in the same environment.

Here they are just development dependencies, so it doesn't matter much.

---

It will take me a while to read all of that. You are welcome to change it as you wish, since this is your project, or wait until I have read through all of https://iscinumpy.dev/post/bound-version-constraints/ and respond.

---

There's no rush so no problem waiting :slightly_smiling_face: Also, mkdocstrings is my project, but I want to you to feel at home maintaining the VBA handler. I'm fine with whatever way you choose to manage your dependencies! So definitely feel free to resolve this conversation and merge :smile: